.github/meta_issue_template/release-1.1.1-plugin-issues-min.md

```diff
@@ -0,0 +1,53 @@
+This is a component issue for release 1.1.1.
+Coming from [release issue 1.1.1](https://github.com/opensearch-project/opensearch-build/issues/870), release version 1.1.1. Please follow the following checklist.
+
+<details><summary>How to use this component issue</summary>
+<p>
+
+## This Component Issue
+This component issue captures the state of the OpenSearch release, on component/plugin level, its assignee is responsible for driving the release of the component.  Please contact them or @mention them on this issue for help. 
+
+## Release Steps
+There are several steps to the release process, components that are behind present risk to the release.  Component owners resolve tasks on this ticket to communicate with the overall release owner.
+
+Steps have completion dates for coordinating efforts between the components of a release; components can start as soon as they are ready far in advance of a future release.
+
+You can find all the corresponding dates of each step in the release issue above.
+
+</p>
+</details>
+
+
+### You can find all the date in above issue 
+### (We make changes to both OpenSearch and OpenSearch-Dashboards for 1.1.1)
```

Comment:
Can we remove this line - if we have nodes we should integrate them into the how to use section?  If we need teams to take different action then we should call it out in the task list below

Reply:
Sure.